lib/test_linker.rb

```diff
@@ -23,7 +23,7 @@ class TestLinker
   DEFAULT_TIMEOUT = 30
 
   # Path the the XMLRPC interface (via the xmlrpc.php file) on the server.
-  DEFAULT_API_PATH = "/lib/api/xmlrpc.php"
+  DEFAULT_API_PATH = "/lib/api/xmlrpc/v1/xmlrpc.php"
```

Comment:
This will not work with earlier versions where the xmlrpc.php is a dir lower. I know later versions use this. We have only 1.9.13 running. What's your idea about supporting different versions?